api/src/controllers/export-data.js

```diff
@@ -93,29 +93,24 @@ module.exports = {
     //    by the following auth check in ctx.district (maybe?)
     //  - Still don't let offline users use this API, and instead refactor the
     //    export logic so it can be used in webapp, and have exports works offline
-    return auth.getUserCtx(req)
+    return auth
+      .check(req, ['can_export_all', ...service.permission(type)])
```

Comment:
You've changed permissions - it used to be `can_export_all` OR can export the specific type, and you've made it AND.

Either create a new `hasAnyPermission` function, or go back to the previous way of checking for `_all` and then if that fails, checking for the specific permission.

Reply:
😩 I my eyes definitely skipped the exclamation mark in this `if`. Thanks, it's fixed